src/ExaMPM_TimeIntegrator.hpp

```diff
@@ -252,8 +250,7 @@ void g2p( const ExecutionSpace& exec_space, const ProblemManagerType& pm,
     Kokkos::Experimental::contribute( k_c, k_c_sv );
 
     // Complete global scatter.
-    pm.scatter( Location::Cell(), Field::Density() );
-    pm.scatter( Location::Cell(), Field::Mark() );
+    pm.scatter( Location::Cell() );
```

Comment:
Likewise, at line 320, 323

pm.scatter( Location::Node() );
pm.gather( Location::Node() ); ??

Reply:
It was Cell before though - maybe I'm misunderstanding